valohai_cli/commands/execution/run/dynamic_run_command.py

```diff
@@ -75,6 +78,7 @@ def __init__(
         :param download_directory: Where to (if somewhere) to download execution outputs (sync mode)
         :param runtime_config: Runtime config dict
         :param runtime_config_preset: Runtime config preset identifier (UUID)
+        :param ssh: Whether opens an interactive ssh connection for debugging
```

Comment:
```suggestion
        :param ssh: Whether to chain to `exec ssh` afterward
```
(to copy the documentation for `watch`)